### DISCUSSION TITLE
Welcome to Geralt Discussions!

### DISCUSSION THREAD

**Discussion by bsod2528** [BSOD2528/Geralt#3] [Announcements]
## 👋 Greetings Dear User!
  We’re using Discussions as a place to connect with other members of our Geralt Community. I hope that you:
  - Feel free to ask queries related to Geralt anytime!
  - Suggest and share ideas to improve.
  - Engage with other community members.
  - Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bsnoot-tim** [comment]
why is geralt so cool?

**Reply by bsod2528** [reply]
ayee thank you very much bsnoot. well, i - uuhmm. its not cool as horus tbh.

**Comment by Zeus432** [comment]
indeed very cool

**Comment by Zeus432** [comment]
tell us your secrets Bsod

**Reply by bsod2528** [reply]
my secret eh - :eyes:

**Reply by Zeus432** [reply]
the secret to making geralt so kewl
